lib/src/widgets/yaru_toggle_button_layout.dart

```diff
@@ -31,7 +31,10 @@ part of 'yaru_toggle_button.dart';
 enum _YaruToggleButtonSlot { leading, title, subtitle }
 
 class _YaruToggleButtonLayout extends RenderObjectWidget
-    with SlottedMultiChildRenderObjectWidgetMixin<_YaruToggleButtonSlot> {
+    with
+        // ignore: deprecated_member_use
+        SlottedMultiChildRenderObjectWidgetMixin<_YaruToggleButtonSlot,
```

Comment:
Maybe extend the `SlottedMultiChildRenderObjectWidget` class instead of still use this deprecated mixin?

Reply:
I tried but ended up with all overwrite methods are wrong
What is this widget. Do you know where it is used?
But I can try to correctly create it later

Reply:
Are you sure?
Because it worked fine in my PR: #767